scripts/create-database.js

```diff
@@ -13,13 +12,13 @@ console.log(chalk.cyan('Creating your FaunaDB Database...\n'))
 if (!process.env.FAUNADB_ADMIN_KEY) {
   console.log(
     chalk.yellow(
-      'Required FAUNADB_SERVER_SECRET enviroment variable not found.'
+      'Required FAUNADB_ADMIN_KEY enviroment variable not found.'
```

Comment:
A failed check for `process.env.FAUNADB_ADMIN_KEY` would accidentally inform the user that `FAUNADB_SERVER_SECRET` was not found.

Reply:
Good catch!  Thank you.

Reply:
No worries! I have actually made a few substantive changes and am trying to get things working smoothly. I would definitely appreciate any feedback on the updated codebase.

I actually can't successfully login. I removed the `FAUNADB_ADMIN_KEY` env variable since it just seemed to proxy `FAUNADB_SERVER_SECRET`, and swapped all references out.

Reply:
And I would love any guidance on how you think we could easily add tests for the API endpoints. (I also plan to fork this and type the functions in TS using `netlify-lambda`, if you have any thoughts on that, whether you'd want to merge that here or would rather it stay its own fork.)

Reply:
I love love love TS.  But I think I would prefer to keep the number of topics for this reference app to a minimum.  I've not tried to add TS in with Netlify functions, but would love to see it.

Reply:
Once the problems here are worked out 😬

Reply:
It's actually super easy. It's all thinly documented, I've been working through this over the last few days trying to get a handle on it.

Basically, the only difference is you do `yarn add -D netlify-lambda`, rename files with `.ts` willy-nilly, and use `netlify-lambda build functions/` to compile to `functions-build/`, which is where you point `netlify.toml`.

`netlify-lambda serve` will basically run `tsc -w` and you run `netlify dev` in a separate terminal and they work concurrently. I will probably send in a PR so `netlify dev` does this automatically over the coming week or so.

Reply:
My whole goal is to combine this example with a few others and use `type-graphql` so we can reproduce this example with just decorated class definitions.